### DISCUSSION TITLE
Option to PREFER browser shortcuts OVER site shortcuts

### DISCUSSION THREAD

**Discussion by codexous** [zen-browser/desktop#3039] [Ideas]
# Sites often override the shortcuts I want to use for the browser. 

For example: I want to use Cmd + S for "toggle sidebar's width," but Discord uses that shortcut already. 
Another example: I want to use Cmd + Shift + S for compact mode toggle, but ChatGPT uses that for _it's_ sidebar.

### How can I use the browser if the shortcuts I rely on stop working on a site-by-site basis? This is game-breaking.

# [Solution 2](https://github.com/zen-browser/desktop/discussions/3039#discussioncomment-11765003)
example.com > page info > permissions > override keyboard shortcuts > "block"

# Solution 1
type `about:config` into URL bar, then search for `permissions.default.shortcuts` and change the value to `2` (default value is `0`)

Keep in mind, both solutions break functionality of some shortcuts on macOS and windows (unsure about linux in my own experience, see: https://github.com/zen-browser/desktop/issues/3031#issue-2661054300)

**Comment by l4kr** [comment]
I cannot seem to be able to use backspace on a lot of websites with this

**Reply by codexous** [reply]
i noticed this too @l4kr :( Not sure what to do about it, i have since reverted

**Reply by watermelvin** [reply]
Hi friends, upon reading https://bugzilla.mozilla.org/show_bug.cgi?id=1445942, I discovered that setting permissions.default.shortcuts in about:config to 3 instead of 2 somehow fixes the issue, allowing zen to retain keyboard shortcut control, while also not breaking the backspace key (at least for me on google docs.)

**Comment by codexous** [comment]
### Secondary solution:

example.com > page info > permissions > override keyboard shortcuts > "block"

This lets you choose web pages in particular to assign the permission to, but it still causes the same issues like esc and undo not working
